src/ASM/LR/ASMLRSpline.h

```diff
@@ -181,6 +181,9 @@ class ASMLRSpline : public ASMbase, public ASMunstruct
   //! \brief Computes the total number of integration points in this patch.
   virtual void getNoIntPoints(size_t& nPt, size_t& nIPt);
 
+  //! \brief Swaps between the first and second projection basis.
+  virtual void swapProjectionBasis();
```

Comment:
The same comment applies here. Has this method relevance for the non-mixed case?

Reply:
OK, maybe you already answered that in the commit message which I read a bit too late ;)

Reply:
see above.